### DISCUSSION TITLE
Reduce duplicate bug reports

### DISCUSSION THREAD

**Discussion by SpareSimian** [AdiAddons/AdiBags#846] [Ideas]
It seems like a lot of users are failing to search existing issues before opening a duplicate of an active one. I suggest adding another item in the bug template under Validation to ask the user if they've searched the issue list. 

I'd also recommend moving the Validation section to the top, so people know they're supposed to disable their addons before starting work on composing a bug report.

**Comment by Cidan** [comment]
In my experience, there's very little we can do to "prevent" users from filing duplicate bugs. We've had multiple people simply ignore the validation altogether, then get upset when we ask them to disable other addons, etc.

Just part of the process in maintaining a project, I suppose.
